roles/pulp_content_guard/README.md

```diff
@@ -36,4 +43,25 @@ Example playbook
             ...
             -----END CERTIFICATE-----
           state: present
+          
+    - role: stackhpc.pulp.pulp_content_guard
+      pulp_url: http://localhost:8080
+      pulp_username: admin
+      pulp_password: "{{ secrets_pulp_admin_password }}"
+      pulp_content_guard_rbac:
+        - name: alex-test-rbac_cg-1
+          description: test-description-edited
+          download_groups:
+            - alex-test-group-1
+            - alex-test-group-2
+          state: present
+        - name: alex-test-rbac_cg-2
+          description: test-description2-edited
+          download_groups:
+            - alex-test-group-2
+        - name: alex-test-rbac_cg-3
+          description: test-description3-edited
+          download_groups:
+            - alex-test-group-1
```

Comment:
Do these get used when state=absent?

Reply:
They do not, I just copy/pasted the example from a test playbook I made with the intention of fixing it up at a later date